frontend/workflows-lib/lib/components/workflow/TasksFlowNode.tsx

```diff
@@ -38,6 +39,9 @@ const TaskFlowNode: React.FC<TaskFlowNodeProps> = ({ data, onNavigate }) => {
         width: "100%",
         height: "100%",
         maxHeight: 100,
+        border: data.highlighted ? "1px solid #202740" : "1px solid #ccc",
```

Comment:
I wonder if using a colour for the border would make it a bit clearer which task is highlighted.